File: algoritmo_e_logica/python/q_1.py
# Utilize o comando de repetição “while”, para solicitar que o usuário informe um número. O programa deve continuar executando até que o usuário digite o número 3, momento em que deverá se encerrar

while True:
    num = int(input("Digite um número (3 para sair): "))
    if num == 3:
        break
    else:
        print(f'Você digitou: {num}\n')
        continue

print('Loop encerrado!')

File: algoritmo_e_logica/javascript/q_1.js
// Utilize o comando de repetição “while”, para solicitar que o usuário informe um número. O programa deve continuar executando até que o usuário digite o número 3, momento em que deverá se encerrar

const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

function perguntarNumero() {
    rl.question('Digite um número (3 para sair): ', (input) => {
        let numero = parseInt(input);

        if (numero === 3) {
            console.log('Saindo do programa...');
            rl.close();
        } else {
            console.log(`Você digitou: ${numero}\n`);
            perguntarNumero(); 
        }
    });
}

perguntarNumero();